spec/facility_spec.rb

```diff
@@ -23,4 +31,35 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe '#register_vehicle' do
+    it 'can register vehicles' do
```

Comment:
These tests could be more readable and robust if you broke them into a few different test cases - one for antique cars, one for electric vehicles, one for basic cars. 

This test only covers one case -- the Cruz. So it doesn't capture all the possible scenarios in your if statement in the register_vehicle method, but we *do* want to test for every scenario in a if statement.